gerritlab/global_vars.py

```diff
@@ -105,3 +141,9 @@ def _get_private_token(
         )
 
     raise SystemExit(f"Unable to find private token for {host}")
+
+
+def _get_default_branch(session, project_url) -> str:
```

Comment:
nit. Sure would be nice to not have to do this every time. What do you think of using the upstream branch ref name as the target branch?

`git rev-parse --symbolic-full-name $(git name-rev --name-only @{u})`

Could fall back to this with a warning message: `Warn: get remote branch from {project_url}, to avoid in future, add an upstream branch: git branch --set-upstream-to=origin/main`

Anyway: optimization. I leave to you.

Reply:
Good idea.  Implemented.